xml/art_sle_amazon_ecs.xml

```diff
@@ -41,21 +41,27 @@
     <para>
       Creating a new ECS cluster in Amazon is accomplished by using the
       <link xlink:href="https://console.aws.amazon.com/ecs/home#/firstRun">First
-      Run Wizard</link>
+      Run Wizard.</link> Alternatively you may use the
+      <emphasis role="strong">aws</emphasis> command line tools to
+      <link xlink:href=http://docs.aws.amazon.com/cli/latest/reference/ecs/create-cluster.html">create</link>
+      the cluster. In the First Run Wizard select
+      <emphasis role="strong">Custom</emphasis> if you want to create a
```

Comment:
Is this a GUI menu item, an option, or a command? Try to use a more specific element (`<guimenu>`, `<option>`, etc.)